src/test/ui/regions/regions-struct-not-wf.rs

```diff
@@ -1,40 +0,0 @@
-// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
-// file at the top-level directory of this distribution and at
-// http://rust-lang.org/COPYRIGHT.
-//
-// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
-// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
-// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
-// option. This file may not be copied, modified, or distributed
-// except according to those terms.
-
-// Various examples of structs whose fields are not well-formed.
-
-// revisions:lexical nll
-
-#![allow(dead_code)]
-#![cfg_attr(nll, feature(nll))]
-
-struct Ref<'a, T> {
-    field: &'a T
```

Comment:
we can keep the original error by using that trait/impl trick I described for `regions-enum-not-wf.rs`